workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSSendSystem.cs

```diff
@@ -8,6 +8,7 @@
 namespace Improbable.Gdk.Core
 {
     [DisableAutoCreation]
+    [AlwaysUpdateSystem]
```

Comment:
we are not injecting anything so it should always run by default

Reply:
Things are injected - `GetComponentGroup(...)` registers the same underlying things as marking a struct as `[Inject]`